service/task.go

```diff
@@ -705,6 +709,13 @@ func (uis *UIServer) taskModify(w http.ResponseWriter, r *http.Request) {
 		}
 		gimlet.WriteJSON(w, projCtx.Task)
 		return
+	case "override_dependencies":
+		err = projCtx.Task.SetOverrideDependencies(authUser.Username())
```

Comment:
This should validate that the user is really an admin, since as it's written a non-admin could successfully make an override_dependencies request to the server.

Reply:
Oops, I made a mental note to handle that but forgot